GoogleForCreators/web-stories-wp
# Animations: Add Tooltip to Disabled Options

## Summary
- Adds tooltips to disabled background animations and directional options
- Updates styling of the directional selector to latest designs https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1923%3A424212

## Relevant Technical Choices
NA

## To-do
Accommodate for bg zoom when we update to designs.

## User-facing changes
Hover over a disabled animation and see a tooltip with a helpful message appear.

## Testing Instructions
Create or go to exisiting story in editor -> add some bg media -> go to add an animation to the media -> see that any disabled options have a tooltip -> click on a pan direction that's not disabled -> go to the directional selector and see that any disabled directions also have a tooltip

---



Fixes #5514
